tutorials/roofit/rf515_hfJSON.js

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Please rename this file to `*.json`. I think the `.js` ending can be misleading for the tutorials directory, which is often checked out by the users. `.js` makes it look like JavaScript :)

Reply:
done